docs/integrations/third-party-libraries.md

```diff
@@ -45,6 +45,7 @@ This can be done using third-party libraries created by the community.
 - [zustand-context](https://github.com/fredericoo/zustand-context) — Create a zustand store in React Context, containing an initial value, or use it in your components with isolated, mockable instances.
 - [zustand-di](https://github.com/charkour/zustand-di) — use react props to init zustand stores
 - [zustand-forms](https://github.com/Conduct/zustand-forms) — Fast, type safe form states as Zustand stores.
+- [zustand-hash-storage](https://github.com/MartinGamesCZ/zustand-hash-storage) - Zustand middleware for saving state into URL hash, b64 encoded (can be configured) and debounce timer.
```

Comment:
```suggestion
- [zustand-hash-storage](https://github.com/MartinGamesCZ/zustand-hash-storage) — Zustand middleware for saving state into URL hash, b64 encoded (can be configured) and debounce timer.
```

Reply:
Sorry for that, didn't see

Reply:
no problem. it feels like we could simply use hyphen. 🤔